### ISSUE TITLE
Solution does not support multiple tenants

### ISSUE THREAD

**Issue by Th4tITGuy** [Acumatica/acumatica-boxstorageprovider#9]
I have tried setting up an instance with multiple tenants, and Box doesn't like that at all. 

User in Company 1 initiates a Box connection
User switches to Company 2
User tries to do anything Box related in Company 2 - all operation are invalid, Box returns an Inner Exception ""

The workaround here is to launch a new private mode session every time, but this is not a viable solution in the long run.

**Comment by gmichaud** [comment]
Did you establish a separate connection to Box through the User Profile screen on both tenants?

**Comment by Th4tITGuy** [comment]
Yes both companies have a connection to Box, this is actually also a problem when you switch from one company already logged in, and try to establish a connection in the second one, it returns an Object reference not set. I had to open the second connection "In Private" mode in Google Chrome to do further testing
